### DISCUSSION TITLE
SASpy error

### DISCUSSION THREAD

**Discussion by ed8484** [sassoftware/saspy#416] [Q&A]
`
[<class 'saspy.sasexceptions.SASIONotSupportedError'>, SASIONotSupportedError('STDIO'), <traceback object at 0x000002905FFECAC0>]`

I get this error after importing saspy
`import saspy`

The error that I received is above. I used Anaconda to install SASpy.
I'm using Windows.
I also did download SAS Integration Technologies Client for Windows.

Now, I only have the free SAS studio/SAS OnDemand for Academics. Is this maybe the problem?

I'm guessing I'm missing an important step.

**Comment by tomweber-sas** [comment]
Hey @ed8484, can you change this to an Issue (I think you can click on something here to do that). just to the right: click create issue from discussion. This is a problem (an Issue) not really a discussion topic.
Either way , I can help you solve this but will need a little more information.
Can you provide the full log from python for what you summitted and al the output? And the config file you are using. I don't recognize this error off the top of my head, so as much info as you can provide will help. It looks like you haven't configures saspy, but I can't really tell without more info.

Thanks,
Tom

**Reply by ed8484** [reply]
Hello,
a. That is the full error and the only code is the import of SASpy
b. No idea about the configuration file..how would I find that out?
